Makefile

```diff
@@ -137,6 +137,25 @@ install:
 	cp ./assets/wall_8K.png ${PREFIX}/share/hyprland
 
 	install -Dm644 -t ${PREFIX}/share/man/man1 ./docs/*.1
+	
+cleaninstall:
```

Comment:
why not
```
make clear
make install
```
?

Reply:
correct me if I'm wrong, but `make install` still copies the desktop file to /usr/share/wayland-sessions

and also, I wanted this task to be global, functional regardless of it's an old installation or freshly built from source

Reply:
oh right yea mb